spotify-artist-viewer/Models/SAArtistTableViewController.m

```diff
@@ -42,12 +41,16 @@ - (void)didReceiveMemoryWarning {
 #pragma mark - Text Change
 
 -(void)searchBar:(UISearchBar *)searchBar textDidChange:(NSString *)searchText{
+  
     SARequestManager *manager = [SARequestManager sharedManager];
```

Comment:
Move this line somewhere that only gets run once, like viewDidLoad. The way this is now you are setting the singleton to your manager property every time the search bar text changes. This will not change the functionality its just cleaner to only set the manager once.